create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+    git push origin $TF_VAR_github_repository_branch
+    ```
+
+## Inspecting the results
+
+After a while the busybox namespace and associated pod should be created and running. Let's see what flux has to say about it first:
+
+```bash
+flux get kustomizations
+```
+
+The output should be something similar to (notice the `busybox` line):
+
+```
+NAME       	READY	MESSAGE                                                        	REVISION                                     	SUSPENDED 
+busybox    	True 	Applied revision: main/fa69f917302bcfd35d2959ebc398b3aa13102480	main/fa69f917302bcfd35d2959ebc398b3aa13102480	False    	
+flux-system	True 	Applied revision: main/fa69f917302bcfd35d2959ebc398b3aa13102480	main/fa69f917302bcfd35d2959ebc398b3aa13102480	False 
+```
+
+Now let's see what Kubernetes has to say. Examine the namespaces first:
+
+```bash
+kubectl get ns
+```
+
+The output should be something similar to (notice the `busybox` line):
+
+```
+NAME              STATUS   AGE
+busybox           Active   30s
+default           Active   26h
+flux-system       Active   26h
+kube-node-lease   Active   26h
+kube-public       Active   26h
+kube-system       Active   26h
+```
+
+Where is our Pod? Let's find out:
+
+```bash
+kubectl get pods -n busybox
+```
+
+The output should be something similar to (notice the `busybox1` line):
+
+```
+NAME       READY   STATUS    RESTARTS   AGE
+busybox1   1/1     Running   0          42s
+```
+
+**Success!**
+
+# Final notes
+
+## Overriding default variables for Terraform
+
+The best way of doing it is to create a `project.tfvars` file. There's a [project.tfvars.sample](project.tfvars.sample) provided in this repo so we can just rename it:
+
+```bash
+cp project.tfvars.sample project.tfvars
+```
+And then fill in the right values for the project - [variables.tf](variables.tf) contains the description for each variable.
+
+Now we're going to use this file to override the desired input variables and update the infrastructure as seen below:
+
+```bash
+terraform plan -var-file="project.tfvars"
+terraform apply -var-file="project.tfvars"
+```
+
+## Testing other Flux CD features
+
+[Flux CD](https://fluxcd.io) supports other interesting controllers as well which can be configured and enabled like:
+ - [Notification Controller](https://fluxcd.io/docs/components/notification) which is specialized in handling inbound and outbount events
+ - [Helm Controller](https://fluxcd.io/docs/components/helm) for managing [Helm](https://helm.sh) chart releases
+ - [Image Automation Controller](https://fluxcd.io/docs/components/image) which can update a Git repository when new container images are available
+
+## Cleaning up (uninstalling) resources
+
+If you want to clean up the allocated resources and destroy all your work then [Terraform](https://www.terraform.io) can handle that for you very easily. It's just a matter of invoking the following command from the directory where this repository was cloned on your local machine:
```

Comment:
```suggestion
If you want to clean up the allocated resources and destroy all your work ,then [Terraform](https://www.terraform.io) can handle that for you very easily. Invoke the following command from the directory where this repository is cloned on your local machine:
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.